enable-vcp-image/enable-vcp-scripts/vcp-daemontset.yaml

```diff
@@ -1,19 +1,27 @@
 ---
-apiVersion: extensions/v1beta1
-kind: ThirdPartyResource
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
We should keep `ThirdPartyResource` to support older builds. I think based on the kubernetes version we should create appropriate objects.